src/test/end-to-end/tests/system-admin/system-admin-audit-log-test.ts

```diff
@@ -1,11 +1,77 @@
+import { generateTestLocation, getCurrentDate, removeTestLocationFile } from '../../shared/shared-functions';
+import { config as testConfig } from '../../../config';
+import { createLocation, uploadPublication } from '../../shared/testingSupportApi';
+import { DateTime } from 'luxon';
+
 Feature('System admin audit log');
 
-Scenario('I as a system admin should be able to view audit log', async ({ I }) => {
+Scenario.skip(
```

Comment:
The skip here should be removed

Reply:
Oops thanks for spotting that. Have removed the 'skip' now :)